### PR TITLE
[R2] Document writeHttpMetadata method on R2Object

### PR DESCRIPTION
**Pull request by KianNH**
https://github.com/cloudflare/workers-types/blob/4d2664ba55a5c4cc6eaa3d4a144de3156ba6ebda/index.d.ts#L1100

Formatting looks good to me: 
image

### CODE REVIEW

**Comment by KianNH** [comment]
Admittedly, the `{{foo}}` tags might not be correct for this - never used them in a PR before. :woozy_face:

File: content/r2/runtime-apis.md [hunk 1]
```diff
@@ -143,6 +143,10 @@ async function handleRequest(request) {
 
   - A map of custom, user-defined metadata associated with the object.
 
+- {{<code>}}writeHttpMetadata(headers{{<param-type>}}Headers{{</param-type>}}){{</code>}} {{<type>}}void{{</type>}}
+
+  -  Retrieves the `httpMetadata` from the R2Object and writes to a Headers object. See [HTTP Metadata](#http-metadata).
```
Comment (deadlypants1973) [inline content/r2/runtime-apis.md]:
```suggestion
  -  Retrieves the `httpMetadata` from the R2Object and writes to a Headers object. Refer to [HTTP Metadata](#http-metadata).
```

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]
thank you as always, kian :)

'refer to' > 'see' for docs

**Comment by KianNH** [comment]
squish squashed

on another note - do you want the code suggestion commits to be squashed or just left as-is?

File: content/r2/runtime-apis.md [hunk 2]
```diff
@@ -143,6 +143,10 @@ async function handleRequest(request) {
 
   - A map of custom, user-defined metadata associated with the object.
 
+- {{<code>}}writeHttpMetadata(headers{{<param-type>}}Headers{{</param-type>}}){{</code>}} {{<type>}}void{{</type>}}
+
+  -  Retrieves the `httpMetadata` from the R2Object and writes to a Headers object. Refer to [HTTP Metadata](#http-metadata).
```
Comment (deadlypants1973) [inline content/r2/runtime-apis.md]:
```suggestion
  -  Retrieves the `httpMetadata` from the `R2Object` and writes to a `Headers` object. Refer to [HTTP Metadata](#http-metadata).
```

**Comment by KianNH** [comment]
image

Didn't realise that the merge into `production` 'squashed' them into a single commit - which was the only reason I was squashing them, so it didn't appear as a handful of separate commits.

TIL

**Review by deadlypants1973 — APPROVED** [review]

**Review by kris-evans — APPROVED** [review]

File: content/r2/runtime-apis.md [hunk 3]
```diff
@@ -143,6 +143,10 @@ async function handleRequest(request) {
 
   - A map of custom, user-defined metadata associated with the object.
 
+- {{<code>}}writeHttpMetadata(headers{{<param-type>}}Headers{{</param-type>}}){{</code>}} {{<type>}}void{{</type>}}
+
+  -  Retrieves the `httpMetadata` from the `R2Object` and writes to a `Headers` object. Refer to [HTTP Metadata](#http-metadata).
```
Comment (lukeed) [inline content/r2/runtime-apis.md]:
```suggestion
  -  Retrieves the `httpMetadata` from the `R2Object` and applies their corresponding HTTP headers to the `Headers` input object. Refer to [HTTP Metadata](#http-metadata).
```

**Review by lukeed — APPROVED** [review]

**Pull request merged**